content/docs/render-props.md

```diff
@@ -328,4 +322,4 @@ class MouseTracker extends React.Component {
 }
 ```
 
-In cases where you cannot bind the instance method ahead of time in the constructor (e.g. because you need to close over the component's props and/or state) `<Mouse>` should extend `React.Component` instead.
+In cases where you cannot define the prop statically (e.g. because you need to close over the component's props and/or state) `<Mouse>` should extend `React.Component` instead.
```

Comment:
Is it not a breach of Dependency Inversion principle? The `Mouse` component cannot possibly know whether the components that will use it will pass over static props or not